### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by Nick-Harvey**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 name: Bug report
-about: Create a report to help us improve
-title: ''
+about: Report a problem
+title: "[BUG]"
 labels: ''
 assignees: ''
```
Comment (griffatrasgo) [inline .github/ISSUE_TEMPLATE/bug_report.md:7]: can we change the dependency verbiage?
FROM:
```
RasgoQL Version [e.g. 22]
```

TO:
```
rasgoql package version [e.g. 1.1.0]
rasgotransforms package version [e.g. 1.1.0]
```

Comment (griffatrasgo) [inline .github/ISSUE_TEMPLATE/bug_report.md:7]: > Describe the bug
> A clear and concise description of what the bug is.

I've seen templates in other packages that ask:
```
Expected Behavior:
Actual Behavior:
```
Is there a place for that in the describe the bug section to help focus the information?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -13,8 +13,5 @@ A clear and concise description of what the problem is. Ex. I'm always frustrate
 **Describe the solution you'd like**
```
Comment (griffatrasgo) [inline .github/ISSUE_TEMPLATE/feature_request.md:13]: > Is your feature request related to a problem? Please describe.
> A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
> 
> Describe the solution you'd like
> A clear and concise description of what you want to happen.
> 
> Describe alternatives you've considered
> A clear and concise description of any alternative solutions or features you've considered.

Are we over-fitting this template to problems? Feels more like a bug report to me. Maybe we can make it more generic?

```
What feature are you requesting?

Are you using a workaround to do it in or outside of the product today?

How important is this feature to your continued use of the package? Can you qualify the value / importance of this feature in any way?
```

Just spitballing...

**Comment by Nick-Harvey** [comment]
Changes have been implmented.

**Review by griffatrasgo — APPROVED** [review]

**Pull request merged**